docs/dev/release_process.rst

```diff
@@ -39,6 +39,31 @@ Create a release branch
 If this is a new major or minor release, you need to make a new branch as described above.
 
 
+Update any translation files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If string interface text has changed, or more complete translations are available, translation files should be updated.
+This is currently done by running the ``make downloadmessages`` command. Following this, the specific files that have been updated with approved translations will need to be added to the repository.
+
+Caveat: by default Crowdin will download all translations, not just approved ones, and will often download untranslated strings also. Do not just add all the files that are downloaded when ``make downloadmessages`` is run, as this will lead to untranslated and poor quality strings being included.
+
+If you need to add a new interface language to Kolibri, please see :ref:`new_language` for details.
+
+Finally, strings for any external Kolibri plugins (like kolibri-exercise-perseus-renderer) should also have been updated, a new release made, and the version updated in Kolibri.
```

Comment:
> should also have been updated

I remember being very confused by the process to do this. I ended up manually downloading files from crowdin (though I forget from where) and then manually copying them over.

Reply:
The latest plugin now has commands to do this, rather than the manual copying approach, will update the documentation in the plugin when I do this.